src/fu-util.c

```diff
@@ -208,6 +216,59 @@ fu_util_prompt_for_device (FuUtilPrivate *priv, GError **error)
 	return g_object_ref (dev);
 }
 
+static gboolean
+fu_util_maybe_enable_automatic (FuUtilPrivate *priv, GPtrArray *remotes, GError **error)
+{
+	guint idx;
+
+	/* TRANSLATORS: Display a message asking every time an update is performed */
+	g_print ("%d.\t%s\n", FU_UTIL_HISTORY_DO_NOTHING, _("Do not upload report(s) at this time, but prompt again for future updates"));
```

Comment:
the translators will have a apoplectic fit if you use `report(s)` -- I think ngettext is what you want.